merged Barometer and Air Pressure, noted archived repos

bradcar opened the pull request:
Merged two sections with the same types of sensors. Put air pressure in the name for those who search for the word pressure.  I added (archived) to the repos that were archived by the original author. 

This issue was brought up in: "two headings with same sensors - Barometer & Pressure #173"

bradcar commented:
One of these is both air and water and I noted it as such

mcauser commented on readme.md:
```diff
@@ -813,24 +813,30 @@ Other places you can look for MicroPython Libraries:
+#### Barometer Air and Water Pressure
 
 * [MicroPython-BMPxxx](https://github.com/bradcar/MicroPython_BMPxxx) - Driver for BMP585, BMP581, BMP390, BMP280 Bosch temperature/pressure sensors.
+* mp-bmp3xx-full](https://github.com/jornamon/mp-bmp3xx-full) - MicroPython driver for the Bosch BMP3xx range of barometric pressure sensors. Includes FIFO support. ⏩
```
Missing an `[` before `mp-bmp3xx-full`

mcauser commented:
Although, technically _liquid_ pressure, `Barometer Air and Water Pressure` reads better.

bradcar commented:
sorry about the missing "[", fixed it. 

Also changed title slightly, while you are right it should be "_liquid_" I'm thinking that most will search for water pressure even if they are thinking about other liquids. In fact I saw a really good build online that used a water pressure sensor covered with silicone oil or mineral oil under flexible nitrile that water-proof it for a submersible open-ocean water pressure sensor.

Updated the additional drivers now marked with "(archieved)"

Should be ready to go now. thanks!

mcauser commented:
Nice one, thanks @bradcar
